.github/workflows/deploy.yml

```diff
@@ -72,18 +70,18 @@ jobs:
         name: Install Python
         with:
           python-version: '3.7'
+      - name: Add msbuild to PATH
+        uses: microsoft/setup-msbuild@v1.0.2
+        if: runner.os == 'Windows'
```

Comment:
I guess we don't need this, as there is no windows gpu build yet.

Reply:
Ah, yeah it's not strictly required here. I was just adding it to every cibuildwheel job blindly and didn't really think about it.